AdminCore.DAL/Models/Event.cs

```diff
@@ -28,7 +28,7 @@ public class Event
     public DateTime LastModified { get; set; }
 
     [ForeignKey("EventId")]
-    public ICollection<EventMessage> EventMessages { get; set; }
+    public IList<EventMessage> EventMessages { get; set; }
```

Comment:
I _think_ this change would be ok. "A navigation property that represents the "many" end of a relationship must return a type that implements ICollection, where T is the type of the object at the other end of the relationship."

IList implements ICollection so I think this should be OK? Just want to make sure this doesn't break the navigation property.

https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/dd468057(v=vs.100)

Reply:
AFAIK I think this is a non braking change. For context, I chose to use a more specific `IList` interface as we should be using `List` to take advantage of its orderedness.